### ISSUE TITLE
[jshow] NEW SITE Colour Contrast Incorrect on the side

### ISSUE THREAD

**Issue by pbwebdev** [joomla/joomla-websites#156]
https://joomla-showcase.octopoos-dev.com/

Colour contrast needs to change to become accesisble 
Colour should be at least this 5d6d6d

![screen shot 2015-11-07 at 4 25 06 pm](https://cloud.githubusercontent.com/assets/3688954/11013707/3243032c-856c-11e5-91cf-cf847490f546.png)

**Comment by MATsxm** [comment]
Hmmm
I really hope it is just for testing as this site is................... a Drupal :wink:

**Comment by oliviernolbert** [comment]
Fixed, thanks.

**Comment by MATsxm** [comment]
@oliviernolbert 
I guess you have not fixed the fact that the site is a Drupal ? :smile:

**Comment by MATsxm** [comment]
Thanks
closed see #498 - reopen if needed
